### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Kaustubh01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,25 +2,21 @@
...
-## Getting Started
-
-The [docs][] folder contains a [getting started guide][] and much more tips about using Unity.
-
 ## Unity Assertion Summary
 
 For the full list, see [UnityAssertionsReference.md][].
 
-### Basic Validity Tests
+### Basic Validity Tests:
```
Comment (Letme) [inline README.md:19]: I would not have `:` in title, sorry.

File: README.md [hunk 2]
```diff
@@ -2,25 +2,21 @@
...
+This project is made to test code targeting microcontrollers big and small.
+This project is a single C file and a pair of headers, allowing you to use this with his existing build setup without too much headache.
+You can use any compiler, and may use most of the existing build systems including Make, CMake, etc.
 If you'd like to leave the hard work to us, you might be interested in Ceedling, a build tool also by ThrowTheSwitch.org.
 
 If you're new to Unity, we encourage you to tour the [getting started guide][].
 
-## Getting Started
-
-The [docs][] folder contains a [getting started guide][] and much more tips about using Unity.
```
Comment (Letme) [inline README.md:17]: Why is the link to getting started guide removed? Commit message does not say much.

File: README.md [hunk 3]
```diff
@@ -2,25 +2,21 @@
 
 __Copyright (c) 2007 - 2021 Unity Project by Mike Karlesky, Mark VanderVoord, and Greg Williams__
 
-Welcome to the Unity Test Project, one of the main projects of ThrowTheSwitch.org.
-Unity Test is a unit testing framework built for C, with a focus on working with embedded toolchains.
+Welcome to Unity Test Project, one of the main projects of ThrowTheSwitch.org.
+Unity Test is a unit testing framework built for C, which focuses on working with embedded toolchains.
```
Comment (nirs) [inline README.md:6]: Both changes looks bad to me. Can you explain why they are needed?

File: README.md [hunk 4]
```diff
@@ -2,25 +2,21 @@
...
 
-This project is made to test code targetting microcontrollers big and small.
-The core project is a single C file and a pair of headers, allowing it to the added to your existing build setup without too much headache.
-You may use any compiler you wish, and may use most existing build systems including Make, CMake, etc.
+This project is made to test code targeting microcontrollers big and small.
+This project is a single C file and a pair of headers, allowing you to use this with his existing build setup without too much headache.
+You can use any compiler, and may use most of the existing build systems including Make, CMake, etc.
```
Comment (nirs) [inline README.md:10]: "targetting" is a spelling error - good change.

Changing "The core" to "This" is changing the semantics, not sure why this change is needed.

Replacing "allowing it to the added to your existing" with "allowing you to use this with his existing" is bad, maybe it should be "allowing you to use this with your existing" but I'm not sure it is better than the original text.

**Review by nirs — COMMENTED** [review]
It will be useful to split the spelling fixes to a separate commit.

I would drop the other changes, or at least explain the issue you are trying to solve.

File: README.md (hunk 3 above)
Comment (mkarlesky) [reply README.md:6]: In fact, per [common grammar rules](http://www.kentlaw.edu/academics/lrw/grinker/LwtaThat_Versus_Which.htm), _that_ without a comma is more correct than _which_ with a common. I think the existing _with_ is quite sufficient (though it does not need a comma in front of it).

**Comment by mvandervoord** [comment]
Portions cherry-picked and merged.
